docs/source/visualisation/experiment_tracking.md

```diff
@@ -1,37 +1,53 @@
-# Set up experiment tracking
+# Experiment tracking in Kedro-Viz
 
-Experiment tracking is the process of saving all machine-learning related experiment information so you can compare different runs.
+Experiment tracking is a way to record all information you need to compare machine-learning experiments and recreate these experiments. All experiment data (parameters, metrics, models, plots and other dataset types) can be searched using a timestamp. View a detailed [Kedro-Viz experiment tracking demo](https://demo.kedro.org/experiment-tracking).
 
-[Kedro-Viz](https://github.com/kedro-org/kedro-viz) supports native experiment tracking from [version 4.1.1](https://github.com/kedro-org/kedro-viz/releases/tag/v4.1.1) onwards. When experiment tracking is enabled in your Kedro project, you will be able to access, edit and [compare your experiments](#access-run-data-and-compare-runs) and additionally [track how your metrics change over time](#view-your-metrics-timeline).
+In the past, Kedro had limited support for this functionality. For example, it was possible to version parameters as part of your codebase with a version control system like `git` and [snapshot models, datasets and plots with Kedro’s dataset versioning capabilities](../data/data_catalog.md#version-datasets-and-ml-models). However, Kedro needed a way to log metrics and capture all this logged data as a timestamped run of an experiment. It also needed a way for users to visualise, discover and compare this data.
+
+We started adding in the missing pieces from [Kedro-Viz `>=4.1.1`](https://github.com/kedro-org/kedro-viz/releases/tag/v4.1.1). When experiment tracking is enabled in your Kedro project, you will be able to access, edit and [compare your experiments](#access-run-data-and-compare-runs) and additionally [track how your metrics change over time](#view-and-compare-metrics-data).
 
 ![](../meta/images/experiment-tracking_demo_small.gif)
 
-Enabling experiment tracking features on Kedro-Viz relies on:
+## When should I use experiment tracking in Kedro?
```

Comment:
I will need help from @stichbury and @NeroOkwa in this new section. Does this content work?

Reply:
From @NeroOkwa: "yeah looks great!"

Reply:
Yep, looks good to me. I've made one suggestion but it's not a huge issue so you can leave as you've edited if it's simpler.